### PR TITLE
Bug 1153885 - remove getContactInfoDisplay R=mrjumpy

### PR DESCRIPTION
**Pull request by mrjumpy**
already run testing and passed it.

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c87366904307cdce27782a7b50525f3450ec5ae0

**Comment by julienw** [comment]
closing in favor of https://github.com/mozilla-b2g/gaia/pull/30093

File: apps/sms/views/shared/js/utils.js [hunk 1]
```diff
@@ -495,7 +495,6 @@
         (contacts) => Utils.basicContact(phoneNumber, contacts)
       );
     },
```
Comment (julienw) [inline apps/sms/views/shared/js/utils.js]: I think you should remove the function, now, right ?

**Comment by julienw** [comment]
arg forget it, I used the wrong PR

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c87366904307cdce27782a7b50525f3450ec5ae0

**Comment by julienw** [comment]
sorry, I closed the wrong PR...

File: apps/sms/views/conversation/js/conversation.js [hunk 2]
```diff
@@ -692,11 +692,14 @@ var ConversationView = {
     // Add recipients and fill+focus the Compose area.
     var recipientPromise;
     if (activity.contact && number) {
-      recipientPromise = Utils.getContactDisplayInfo(findByAddress, number)
-        .then(function onData(data) {
-          data.source = 'contacts';
-          return data;
-        });
+      findByAddress(phoneNumber).then((contacts) => {
```
Comment (julienw) [inline apps/sms/views/conversation/js/conversation.js]: here, this is not `phoneNumber`, this is `number` :)

Comment (julienw) [inline apps/sms/views/conversation/js/conversation.js]: note that if you install jshint locally, it should tell it to you once you configure it with your text editor :)

Comment (julienw) [inline apps/sms/views/conversation/js/conversation.js]: you also need to assign the result of this line to `recipientPromise` just like the previous code did

File: apps/sms/views/conversation/js/conversation.js [hunk 3]
```diff
@@ -692,11 +692,14 @@ var ConversationView = {
...
+        return Object.assign(
+          Utils.basicContact(phoneNumber, contacts),
+          { 
+            source: 'contacts' 
+          }
```
Comment (julienw) [inline apps/sms/views/conversation/js/conversation.js]: nit: you can write on one line, I think it's more readable in this case: `{ source: 'contacts' }`

**Pull request closed without merging**